src/models/ballot.ts

```diff
@@ -11,6 +11,15 @@ export enum V2BallotState {
   'Failed' = 2,
 }
 
+export enum V4ApprovalStatus {
```

Comment:
Do u wanna just add a todo to add this to the sdk? should be there imo

Reply:
Yep will do. Do you want me to start adding stuff to the SDK? I'm not sure how to atm but would love to learn